Dockerfile

```diff
@@ -154,7 +154,7 @@ USER dockerdev
 
 RUN \
   if [ "$(ls /tmp)" ]; then ls /tmp; false; fi \
-  && stack install --resolver lts-10.10 sbp \
+  && stack install --resolver lts-23.7 sbp \
```

Comment:
Hey, I think I tried it like this in the past, but it didn't work. I remember you did it like [this](https://github.com/swift-nav/libsbp/commit/c9df176a2c930b19d5754128e77640d8069b4440) previously. Can you double-check?

Reply:
From what I understand, the `Dockerfile` explicitly states what version it wants to use to build the SBP version, where as in the [Haskell test build](https://github.com/swift-nav/libsbp/blob/9daf121f29351f5b91684cbc1b81268b845fd351/.github/workflows/haskell.yaml#L49) it uses whatever is in that `haskell/stack.yaml` file. I don't know why there is a difference there.

Reply:
The problem strangely that I'm seeing with my PR is that Haskell and NodeJS are all working, but its python that keeps failing now. I've rolled back the Haskell from `23.7` to `14.9` and it still fails. 

![image](https://github.com/user-attachments/assets/a4eec9df-86f5-4392-9ad9-2ced5604f6d6)

Reply:
I've since reverted this change and tried to unify the Haskell version to `lts-14.9`